### ISSUE TITLE
Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by weihaosky** [graphdeco-inria/gaussian-splatting#501]
Hi, the Network Viewer works well with `./<SIBR install dir>/bin/SIBR_remoteGaussian_app`.
However, when I use the Real-Time Viewer with `./<SIBR install dir>/bin/SIBR_gaussianViewer_app -m <path to trained model>`, the segmentation fault error occurs, as
```
[SIBR] --  INFOS  --:   Initialization of GLFW
[SIBR] --  INFOS  --:   OpenGL Version: 4.6.0 NVIDIA 525.147.05[major: 4, minor: 6]
Number of input Images to read: 185
Number of Cameras set up: 185
[SIBR] --  INFOS  --:   Error: can't load mesh '/home/weihao/disk16/datasets/mip360_v2/garden/sparse/0/points3d.txt.
[SIBR] --  INFOS  --:   Error: can't load mesh '/home/weihao/disk16/datasets/mip360_v2/garden/sparse/0/points3d.ply.
[SIBR] --  INFOS  --:   Error: can't load mesh '/home/weihao/disk16/datasets/mip360_v2/garden/sparse/0/points3d.obj.
LOADSFM: Try to open /home/weihao/disk16/datasets/mip360_v2/garden/sparse/0/points3D.bin
Num 3D pts 138766
[SIBR] --  INFOS  --:   SfM Mesh '/home/weihao/disk16/datasets/mip360_v2/garden/sparse/0/points3d.txt successfully loaded.  (138766) vertices detected. Init GL ...
[SIBR] --  INFOS  --:   Init GL mesh complete 
Segmentation fault (core dumped)
```

Anyone could help me? Many thanks!

**Comment by Coder-ZZY** [comment]
I had the same problem, but I can't run both SIBR_remoteGaussian_app and SIBR_gaussianViewer_app

**Comment by Coder-ZZY** [comment]
[SIBR] --  INFOS  --:   Initialization of GLFW
Segmentation fault (core dumped)
but mine happens in Initialization of GLFW

**Comment by ZhentaoHuang** [comment]
I am having the same problem here. I am on Ubuntu 20.04. With --no_interop can run it but it is slower than expected.

**Comment by leoShen917** [comment]
I am also having the same problem and Ubuntu 20.04. Has anyone solved this problem, please?

**Comment by abysee** [comment]
I tried debugging using gdb and after back trace it located the problem at SIBR_viwers/src/projects/remote/renderer/RemotePointView.cpp:266 in which it looks like 
```
for (const auto& pair : globalMetricsDict) {
            ImGui::Text("%s: %.3f", pair.first.c_str(), pair.second);
      } 
```
. After commented these lines and rebuilt the SIBR program, the problem disappeared. This problem have bothered me for a long time and even though I solved it I'm still not sure what kind of fault data pointer have led to this problem. Hope it helps solve your issue.
